README.md

```diff
@@ -23,7 +23,6 @@
 ## 📖 About the Project <a name="about-project"></a>
 
 Welcome to Budget, your personal finance management application built with Ruby on Rails. This app helps you manage your expenses efficiently by organizing transactions into categories and providing insightful financial data. Keep track of your spending, create new categories, and add transactions with ease.
-
 This project is inspired by the original design idea by Gregoire Vella on Behance, and it is used under the Creative Commons license.
```

Comment:
- [ ] The deployed version of your app that you provided in the README file is not working properly and instead of showing the app pages, it shows a blank page with `not found` text in it. Please fix that.

![image](https://github.com/hafiz1379/Budget-app/assets/68796232/8fc9fbe7-37be-48ae-814c-fd717a2df2b9)

Reply:
Fixed.